src/app/pages/StoryPage/topStories.json

```diff
@@ -0,0 +1,122 @@
+[
```

Comment:
Maybe we should indicate these are temporary pieces of fixture data, either in the filename or we move them to a folder?

Reply:
I have added a task to delete the fixture data files, to the issue for [initial solution for data for F&A component](https://github.com/bbc/simorgh/issues/5765). I assume a similar issue will be derived for Top Stories after this PR.